.travis.yml

```diff
@@ -81,13 +81,16 @@ script:
       export CCACHE_CPP2=yes;
     fi
   - mkdir build &&
+    travis_wait 60 cmake
+        ${BACKEND:+-DKokkos_ENABLE_${BACKEND}=On}
+        -DKokkos_ENABLE_TESTS=On
+        ${CMAKE_BUILD_TYPE:+-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}}
+        -DCMAKE_INSTALL_PREFIX=
+        -DNUM_PROC=2
+        -DSITE=Travis
+        -DCTEST_ARGS=-VV
```

Comment:
This will blow up the log on Travis.  We had purposefully chosen to only print on failure.

Reply:
can do -V or output on failure or whatever. You'll probably never look at the Travis log unless it fails to submit to cdash. The entire log of the test whether it passes or fails will be there on the dashboard